plexupdate.sh

```diff
@@ -517,6 +520,8 @@ if [ "${AUTOSTART}" = "yes" ]; then
 		service plexmediaserver start
 	elif [ -x /etc/init.d/plexmediaserver ]; then
 		/etc/init.d/plexmediaserver start
+	elif [ "${DISTRO}" = "synology" ]; then
+		synopkg start "Plex Media Server"
```

Comment:
Don't forget to also update `extras/installer.sh` to enable `AUTOSTART` when installing on Synology.

Reply:
do you think AUTODELETE should be enabled by default if the user chooses to AUTOINSTALL?

Reply:
Hmm. I personally prefer to keep the old copies around just as a way to review my update history, but I think any reasonable person (i.e. not me) would probably expect updates to happen without littering up your hard drive, so I think that's a great idea actually.